common/persistence/sql/sqlExecutionManager.go

```diff
@@ -441,9 +441,7 @@ func (m *sqlExecutionManager) updateWorkflowExecutionTx(
 				newWorkflow.ExecutionInfo.CloseStatus,
 				startVersion,
 				lastWriteVersion); err != nil {
-				return &types.InternalServiceError{
-					Message: fmt.Sprintf("UpdateWorkflowExecution: failed to continue as new current execution. Error: %v", err),
-				}
+				return err
```

Comment:
This is to avoid duplicate error conversion which can lead to errors such as `InternalServiceError{Message: UpdateWorkflowExecution: failed to update current execution. Error: ServiceBusyError{Message: updateCurrentExecution operation failed.  Error: failed to update rows: Full rate-limiter queue}}`